### PR TITLE
✏️ Fix ping parameters annotation in Config class

### PR DESCRIPTION
**Pull request by Kludex**
Hard to find :sweat_smile:

### CODE REVIEW

**Comment by graingert** [comment]
https://github.com/aaugustin/websockets/blob/361f38066622636eff8d9e0d3d071e93e8e02b05/src/websockets/legacy/protocol.py#L1097 actually pretty sure they should be optional to allow for disabling ping?

File: uvicorn/config.py [hunk 1]
```diff
@@ -145,8 +145,8 @@ def __init__(
         http: Union[Type[asyncio.Protocol], HTTPProtocolType] = "auto",
         ws: Union[Type[asyncio.Protocol], WSProtocolType] = "auto",
         ws_max_size: int = 16 * 1024 * 1024,
-        ws_ping_interval: int = 20,
-        ws_ping_timeout: int = 20,
+        ws_ping_interval: float = 20,
```
Comment (graingert) [inline uvicorn/config.py]: Eventually passed here:

https://github.com/aaugustin/websockets/blob/361f38066622636eff8d9e0d3d071e93e8e02b05/src/websockets/legacy/server.py#L994-L995

**Review by graingert — CHANGES_REQUESTED** [review]
I think these should be Optional [float]

**Review by graingert — APPROVED** [review]

**Comment by Kludex** [comment]
@graingert True! Thanks for the review! I'm going to push the fix later on.

**Pull request merged**